rules/integrations/kubernetes/execution_user_exec_to_pod.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2022/05/17"
+maturity = "production"
+updated_date = "2022/05/17"
+integration = "kubernetes"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a user attempt to establish a shell session into a pod using the 'exec' command. Using the 'exec' command in a pod allows a user to establish a temporary shell session and execute any process/commands in the pod. An adversary may call bash to gain a persistent interactive shell which will allow access to any data the pod has permissions to, including secrets. 
+"""
+false_positives = [
+    """
+    An administrator may need to exec into a pod for a legitimate reason.
```

Comment:
Can we expand this at all (maybe a specific example)

Reply:
Expanded and included examples from the official documentation.